python-sdk/example_dags/example_load_file.py

```diff
@@ -221,4 +222,8 @@
     )
     # [END load_file_example_17]
 
+    # [START load_file_example_18]
```

Comment:
@sunank200 Also should we move this to tests instead? and/or convert to Unit test instead? Since we are planning to move away from integration tests as much as possible.